contracts/libraries/FullMath.sol

```diff
@@ -6,7 +6,7 @@ pragma solidity 0.6.12;
 // taken from https://medium.com/coinmonks/math-in-solidity-part-3-percents-and-proportions-4db014e080b1
 // license is CC-BY-4.0
 library FullMath {
```

Comment:
Has the FullMath library ever been used in production?
Can we do without it to reduce complexity of the changes to the Cauldron contracts?